runtime/src/snapshot_utils.rs

```diff
@@ -684,30 +687,6 @@ pub fn read_full_snapshot_slot_file(bank_snapshot_dir: impl AsRef<Path>) -> io::
 
 /// Writes files that indicate the bank snapshot is loadable by fastboot
 pub fn mark_bank_snapshot_as_loadable(bank_snapshot_dir: impl AsRef<Path>) -> io::Result<()> {
-    // Mark this directory complete. Used in older versions to check if the snapshot is complete
-    // Never read in v3.1, can be removed in v3.2
-    let state_complete_path = bank_snapshot_dir
-        .as_ref()
-        .join(SNAPSHOT_STATE_COMPLETE_FILENAME);
-    fs::File::create(&state_complete_path).map_err(|err| {
-        IoError::other(format!(
-            "failed to create file '{}': {err}",
-            state_complete_path.display(),
-        ))
-    })?;
-
-    // Write the storages flushed file. Used in older versions to check if the snapshot is complete
-    // Read in v3.1 for backwards compatibility, can be removed in v3.2
```

Comment:
Why did we say that we couldn't remove this until v3.2 yet we're removing it now? (Same as comment above too.)

Reply:
Talked with Rory offline. We're still reading the files; this change is just to not write them anymore.